assets/prototype/domain/eventEditor/data/queries/datetimes/test/data.ts

```diff
@@ -42,10 +43,10 @@ export const nodes: Datetime[] = [
 		isDeleted: false,
```

Comment:
We should consider renaming this field to `isTrashed`

Reply:
This is related to this issue: https://github.com/eventespresso/event-espresso-core/issues/2089